database/warehouse/events/unit_died_event.py

```diff
@@ -38,6 +36,59 @@ class unit_died_event(Base):
     info_id = Column(Integer, ForeignKey("replay.info.primary_id"))
     info = relationship("info", back_populates="unit_died_events")
 
-    player_id = Column(Integer, ForeignKey("replay.player.primary_id"))
```

Comment:
Normalize table. Keep references to objects involved in event. Ownership belongs on object table.